vnc.html

```diff
@@ -274,9 +274,9 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br>VNC</h1>
     <div class="noVNC_center">
         <div id="noVNC_connect_dlg">
             <div class="noVNC_logo" translate="no"><span>no</span>VNC</div>
-            <div id="noVNC_connect_button"><div>
+            <div id="noVNC_connect_box"><button>
```

Comment:
Is the outer div even needed?

Reply:
No, fixed in 05baf14256a4640e15cda63cc2ac576f44482775